docs/platform-integration/essentials/preferences.md

```diff
@@ -1,107 +1,141 @@
+- <xref:System.String>
+- <xref:System.DateTime>
+
+Values of `DateTime` are stored in a 64-bit binary (long integer) format using two methods defined by the `DateTime` class:
+
+- The [`ToBinary`](xref:System.DateTime.ToBinary) method is used to encode the `DateTime` value.
+- The [`FromBinary`](xref:System.DateTime.FromBinary(System.Int64)) method decodes the value.
+
+See the documentation of these methods for adjustments that might be made to decoded values when a `DateTime` is stored that isn't a Coordinated Universal Time (UTC) value.
 
 ## Get started
 
 [!INCLUDE [get-started](includes/get-started.md)]
 
-## Using Preferences
-
 [!INCLUDE [essentials-namespace](includes/essentials-namespace.md)]
 
-To save a value for a given _key_ in preferences:
+## Set preferences
+
+Preferences are set by calling the `Preferences.Set` method, providing the key and value:
 
 ```csharp
-Preferences.Set("my_key", "my_value");
+// Set a string value:
+Preferences.Set("first_name", "John");
+
+// Set an numerical value:
+Preferences.Set("age", 28);
+
+// Set a boolean value:
+Preferences.Set("has_pets", true);
 ```
 
-To retrieve a value from preferences or a default if not set:
+## Get preferences
+
+To retrieve a value from preferences, you pass the key of the preference, followed by the default value when the key doesn't exist:
 
 ```csharp
-var myValue = Preferences.Get("my_key", "default_value");
+string firstName = Preferences.Get("first_name", "Unknown");
+int age = Preferences.Get("age", -1);
+bool hasPets = Preferences.Get("has_pets", false);
 ```
 
-To check if a given _key_ exists in preferences:
+## Check for a key
+
+It may be useful to check if a key exists in the preferences or not. Even though `Preferences.Get` has you set a default value when the key doesn't exist, there may be cases where the key existed, but the value of the key matched the default value. So you can't rely on the default value as an indicator that the key doesn't exist. Use the `Preferences.ContainsKey` method to determine if a key exists:
 
 ```csharp
 bool hasKey = Preferences.ContainsKey("my_key");
 ```
 
-To remove the _key_ from preferences:
+## Remove one or all keys
+
+Use the `Preferences.Remove` method to remove a specific key from preferences:
 
 ```csharp
-Preferences.Remove("my_key");
+Preferences.Remove("first_name");
 ```
 
-To remove all preferences:
+To remove all keys, use the `Preferences.Clear` method:
 
 ```csharp
 Preferences.Clear();
 ```
 
-> [!TIP]
-> The above methods take in an optional `string` parameter called `sharedName`. This parameter is used to create additional containers for preferences which are helpful in some use cases. One use case is when your application needs to share preferences across extensions or to a watch application. Please read the platform implementation specifics below.
+## Shared keys
+
+<!-- TODO: What is the difference between an extension and a different app? I'm confused (and new to .NET MAUI) -->
```

Comment:
This is all about iOS, specifically app extensions (https://developer.apple.com/app-extensions/) and watchOS apps (which can be linked to iOS apps, hence the need for shared preferences). I think the sentence is (largely) fine, but may need revisiting later when we find out if Essentials will still support watchOS.